src/transformers/configuration_utils.py

```diff
@@ -98,6 +98,18 @@ def __init__(self, **kwargs):
                 logger.error("Can't set {} with value {} for {}".format(key, value, self))
                 raise err
 
+    @property
+    def num_labels(self):
+        return self._num_labels
+
+    @num_labels.setter
+    def num_labels(self, num_labels):
+        self._num_labels = num_labels
```

Comment:
this is not perfect b/c it serializes a `_num_labels` in the config.json files (which still works as intended, but a bit messy)

Reply:
Yes, I agree. It is backwards compatible though with previous `num_labels` fields.